spec/predicates/human-review/vcs.md

```diff
@@ -0,0 +1,103 @@
+# Predicate type: VCS Code Reviews
+
+Type URI: (tentative) https://in-toto.io/attestation/human-review/vcs/v0.1
+
+Version: 0.1.0
+
+## Purpose
+
+This attestation type is used to describe code review approvals of the simplest
+kind. The general expectation is that reviews are tied to specific changes in a
+version control system (VCS) such as Git.
+
+## Use Cases
+
+Changes to source code are reviewed by other members of the development team.
+The review may be performed using systems like Gerrit or via reviewing
+mechanisms built into version control forges such as GitHub and GitLab.
+
+## Prerequisite
+
+Understanding of the
+[in-toto attestation specification](https://github.com/in-toto/attestation).
+
+## Model
+
+Typically, software development processes incorporate some type of code review.
+Other members of the development team are expected to study the changes
+proposed and approve them, confirming they meet various expectations pertaining
+to code quality, functionality, and so on. Therefore, the general expectation
+is that this attestation is generated as a confirmation, an approval of some
+specific change, and is performed by developers in the supply chain who have
+code review responsibilities.
+
+## Schema
+
+```json
+{
+    "_type": "https://in-toto.io/Statement/v1",
+    "subject": [
+        {
+            "digest": { "gitCommit": "<HASH>" }
+        },
+    ],
+    "predicateType": "http://in-toto.io/attestation/human-review/vcs/v0.1",
+    "predicate": {
+        "result": "approved/changes-required/comment",
+        "reviewLink": "https://github.com/<>",
+        "timestamp": "<TIMESTAMP>"
+        "reviewer": "<STRING>"
```

Comment:
Should reviewer be a list or is the idea there would be one attestation per reviewer?

Reply:
I think attestation per reviewer, that way we can plug into the threshold semantics directly for policies like two party review.

Reply:
I think there's interesting scope here for making the vcs one:
1. approve only
2. include 1+ reviewer details.

This addresses the signed-by-github point above. For human created ones, we can use the generic one. Thoughts? I could even split vcs into github and gerrit to show off how they'd differ, perhaps?